crates/holochain_state/src/source_chain.rs

```diff
@@ -1289,7 +1289,7 @@ pub async fn dump_state(
             let published_ops_count = txn.query_row(
                 "
                 SELECT COUNT(DhtOp.hash) FROM DhtOp
-                LEFT JOIN Action ON DhtOp.action_hash = Action.hash
+                JOIN Action ON DhtOp.action_hash = Action.hash
```

Comment:
These lefts just keep coming and going : )

Reply:
I hope this is the last one! All `LEFT JOIN Action`s must go.